mlflow/utils/file_utils.py

```diff
@@ -483,3 +485,54 @@ def _handle_readonly_on_windows(func, path, exc_info):
         raise exc_value
     os.chmod(path, stat.S_IWRITE)
     func(path)
+
+
+_TMP_DIR = None
```

Comment:
Can we use a more descriptive name?

Reply:
+1

Reply:
I think an inline comment above the variable name is a better choice. It makes sense that this is called `_TMP_DIR` because it's referenced by `get_or_create_tmp_dir`.